string Device
string Command
bool GPIO_on
